Chrome-launcher typescript compiler, can't find name

After adding chrome-launcher as a module to the [pwmetrics](https://github.com/paulirish/pwmetrics) project I came across this typescript issue:
```
> pwmetrics@2.0.1 build /home/pedro.cardoso/dev/pwmetrics
> tsc

lib/index.ts(36,13): error TS2304: Cannot find name 'chromeLauncher'.
lib/index.ts(146,33): error TS2503: Cannot find namespace 'chromeLauncher'.
```

The respective code is: 
```js
const chromeLauncher = require('chrome-launcher');
....

 async launchChrome(): Promise<chromeLauncher.LaunchedChrome|Error> {
    try {
      console.log(messages.getMessage('LAUNCHING_CHROME'));
      this.launcher = await chromeLauncher.launch({
        port: this.flags.port,
        chromeFlags: this.flags.chromeFlags
      });
      this.flags.port = this.launcher.port;
    } catch(error) {
      await this.killLauncher();
      return error;
    }
  }
```

If I add the project as a js module (not in the docs I know) I get the following:
```
lib/index.ts(5,38): error TS7016: Could not find a declaration file for module 'chrome-launcher'. '/home/pedro.cardoso/dev/pwmetrics/node_modules/chrome-launcher/chrome-launcher.js' implicitly has an 'any' type.
```

I import the module as: 
```js
import {chromeLauncher, LaunchedChrome} from 'chrome-launcher';
....

async launchChrome(): Promise<LaunchedChrome|Error> {
    try {
      console.log(messages.getMessage('LAUNCHING_CHROME'));
      this.launcher = await chromeLauncher.launch({
        port: this.flags.port,
        chromeFlags: this.flags.chromeFlags
      });
      this.flags.port = this.launcher.port;
    } catch(error) {
      await this.killLauncher();
      return error;
    }
  }
```

__System details__: 
Ubuntu 16.04 LTS
Node: V8.0.0
NPM: V5.0.0

__How to reproduce__:
Trying to merge LH.v2 to pwmetrics and add the chrome-launcher dependency :)

Thanks for the time.

Thanks for looking into this @samccone, i've tried your suggestion but I still get the message:
 
```
lib/index.ts(5,22): error TS7016: Could not find a declaration file for module 'chrome-launcher'. '/home/pedro.cardoso/dev/pwmetrics/node_modules/chrome-launcher/chrome-launcher.js' implicitly has an 'any' type.
```

Here is the travis build: https://travis-ci.org/paulirish/pwmetrics/builds/242740991
and the relevant source code (its incapsulated in index.ts): https://github.com/paulirish/pwmetrics/pull/120/files#diff-13b5b151431c7e7a17f273559ed212d5

I noticed that chrome-launcher is exported as a standard javascript project, and that it doesn't have a explicit types files, could either of these be the cause of the problems? Or can I not use the chrome-launcher types in pwmetrics?

--- a/chrome-launcher/index.ts
+++ b/chrome-launcher/index.ts
@@ -1,0 +1,1 @@
+export * from './chrome-launcher';

--- a/chrome-launcher/package.json
+++ b/chrome-launcher/package.json
@@ -1,6 +1,6 @@
 {
   "name": "chrome-launcher",
-  "main": "chrome-launcher.js",
+  "main": "index.js",
   "scripts": {
     "build": "tsc",
     "dev": "tsc -w",

